staging/src/k8s.io/apiserver/pkg/server/filters/longrunning.go

```diff
@@ -32,6 +33,9 @@ func BasicLongRunningRequestCheck(longRunningVerbs, longRunningSubresources sets
 		if requestInfo.IsResourceRequest && longRunningSubresources.Has(requestInfo.Subresource) {
 			return true
 		}
+		if !requestInfo.IsResourceRequest && strings.HasPrefix(requestInfo.Path, "/debug/pprof/") {
+			return true
+		}
```

Comment:
this looks right. @deads2k has a better overview about the current mux setup for this kind of routes.